packages/apps/src/NodeInfo.tsx

```diff
@@ -16,8 +16,15 @@ import translate from './translate';
 
 type Props = I18nProps & {};
 
+export function isTestChain(chain?: string): boolean {
+  if (typeof chain === 'undefined') return false;
+  const re = new RegExp("(dev|loc)", "i");
+  const match = re.test(chain.toLowerCase());
+  return match ? true : false;
```

Comment:
Space before `const` as well as after - can be grouped, but just leave some visual indication that the declaration block is different.